ADOAccess/Program.cs

```diff
@@ -1,16 +1,16 @@
-﻿using System.Data.SqlClient;
+﻿using Microsoft.Data.SqlClient;
 
 Console.WriteLine("Querying for blogs");
 
-using var conn = new SqlConnection(@"Server=(localdb)\mssqllocaldb;Database=master;Trusted_Connection=True;");
+await using var conn = new SqlConnection(@"Server=(localdb)\mssqllocaldb;Database=master;Trusted_Connection=True;");
```

Comment:
Adding `await` before `using` means that when the instance (the connection) is disposed (goes out of scope), it's disposed asynchronously instead of synchronously (DisposeAsync instead of Dispose). If disposing involves any I/O, this ensures that the thread doesn't block.